linux_os/guide/services/obsolete/nis/package_ypserv_removed/rule.yml

```diff
@@ -19,11 +19,13 @@ references:
     stigid@rhel6: RHEL-06-000220
     srg@rhel6: SRG-OS-000095
     nist@rhel6: CM-7
+    nist-csf@rhel6: PR.IP-1,PR.PT-3
```

Comment:
How come RHEL6  and RHEL7 would have different mappings?

Reply:
Because in the 800-53 to CSF mapping, each 800-53 control can map to multiple CSF controls hence the different mappings.